packages/plugins/whisper/src/whisperClient.js

```diff
@@ -1,12 +1,11 @@
 import { __ } from 'embark-i18n';
 const async = require('async');
-const GethMiner = require('./miner');
 const semver = require('semver');
 const constants = require('embark-core/constants');
 
 const DEFAULTS = {
   "BIN": "geth",
-  "VERSIONS_SUPPORTED": ">=1.9.7",
+  "VERSIONS_SUPPORTED": ">=1.8.14",
```

Comment:
What's the reason for dropping the officially min supported version back to `1.8.14`, i.e. as opposed to keeping it matched up with `1.9.7` in [packages/plugins/geth/src/gethClient.js#L12](https://github.com/embark-framework/embark/blob/refactor/whisper-node-arch/packages/plugins/geth/src/gethClient.js#L12)?

Reply:
This section of code is marked by GitHub as *outdated* because of various changes nearby to the line I had/have a question about. Going to mark as resolved but will also create a new comment with the same question.